dev/com.ibm.ws.jdbc_fat/test-applications/basicfat/src/basicfat/DataSourceTestServlet.java

```diff
@@ -215,20 +215,36 @@
      * Create the default table used by the tests.
      */
     private void createTable(DataSource ds) throws SQLException {
-        Connection con = ds.getConnection();
-        try {
-            Statement st = con.createStatement();
-            try {
-                st.executeUpdate("drop table cities");
-            } catch (SQLException x) {
+        try (Connection con = ds.getConnection()) {
+            //See if cities table was already created
+            DatabaseMetaData md = con.getMetaData();
+            boolean tableFound = false;
+            ResultSet rs = md.getTables(null, null, "CITIES", null); // Some DB providers require the table name in all upper case
+            while (rs.next()) {
+                if (rs.getString("TABLE_NAME").equalsIgnoreCase("cities")) {
+                    tableFound = true;
+                }
+            }
+
+            if (!tableFound) {
+                rs = md.getTables(null, null, "cities", null); // Other DB providers require the table name in all lower case
+                while (rs.next()) {
+                    if (rs.getString("TABLE_NAME").equalsIgnoreCase("cities")) {
+                        tableFound = true;
+                    }
+                }
+            }
+
+            try (Statement st = con.createStatement()) {
+                if (tableFound) {
+                    st.executeUpdate("drop table cities");
+                }
+                String dbProductName = md.getDatabaseProductName().toUpperCase();
+                if (dbProductName.contains("IDS") || dbProductName.contains("INFORMIX")) // Informix JCC and JDBC
```

Comment:
We no longer test against Informix so this logic can be cut out.

Reply:
There are multiple places where Informix is referenced.  Should I remove them all in DataSourceTestServlet, or just the one you highlighted?

Reply:
Yeah we can remove anything that references Informix in this test bucket.  But, that can be done in a future commit.